### ISSUE TITLE
Using curry with bind cause a typescript error

### ISSUE THREAD

**Issue by ArielGueta** [lodash/lodash#3626]
I'm trying to use the bind method with curry, but it gives me a type error. 

```
const curried = curry(this.method.bind(this));
const getSomething = curried(a, b);
```

Getting TS error from `getSomething` - expected 0-1 arguments but got 2.

**Comment by falsyvalues** [comment]
I think you have answer from TS compiler `expected 0-1 arguments but got 2.`.

**Comment by ArielGueta** [comment]
I don't understand your answer. The problem is with lodash typing. When I am not using the bind method it's not complaining. 

```
const curried = curry(this.method);
const getSomething = curried(a, b);
```

**Comment by falsyvalues** [comment]
TypeScript typings aren't maintained by Lodash team :arrow_forward:  [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped).
